src/Operations.jl

```diff
@@ -2425,6 +2425,16 @@ function print_status(env::EnvCache, old_env::Union{Nothing,EnvCache}, registrie
             printpkgstyle(io, :Info, "Some packages have new versions but compatibility constraints restrict them from upgrading.$tip", color=Base.info_color(), ignore_indent)
         end
     end
+    if outdated && !no_packages_upgradable
+        printpkgstyle(io, :Info, """
+            If a package is marked by $upgradable_indicator but not upgrading, the resolver may \
+            have other less optimal solutions that do so.
```

Comment:
Suggestion: "If a package is marked by $upgradable_indicator but not upgrading, it was likely restricted by a secondary dependency" (right? isn't that the only case where this would show up?)

Reply:
No it could be that the resolver has multiple solutions, but the one it went for was the one that changed the least. In that case force adding a version should work but downgrade more

Reply:
Is that due to the `preserve` argument defaulting to `all`?